### ISSUE TITLE
Ability to teleport mouse to focused window's center

### ISSUE THREAD

**Issue by marcelohdez** [martinhjartmyr/gnome-shell-extension-focus-changer#22]
Hey! It would be great if this extension could also teleport the mouse cursor to the center of the focused window like tiling window managers :D

**Comment by martinhjartmyr** [comment]
Yes, it's a good idea. Probably easy to fix. It needs a toggle in the settings so you can opt in/out as well.

**Comment by gcem** [comment]
This extension does exactly that: https://extensions.gnome.org/extension/4642/mouse-follows-focus/
